wp/lib/bap_wp/src/environment.mli

```diff
@@ -83,6 +89,9 @@ type exp_cond = t -> Bap.Std.Exp.t -> cond_type option
     - the architecture of the binary
     - the option to freshen variable names
     - the option to use all input registers when generating function symbols at a call site
+    - the range of addresses of the stack
```

Comment:
The *concrete* range of addresses. We might want to make these more abstract at some point, but let's keep it for now.